scripts/unit/binary/util/test.json

```diff
@@ -0,0 +1,3 @@
+{
+  "test": "true"
+}
```

Comment:
? this seems like it was previously mocked by mock-fs but now isn't?

Reply:
Ah this was just something for debugging/testing that I forgot to remove. I'll get rid of that